src/item.ts

```diff
@@ -1,6 +1,7 @@
 export interface JSONWarehouseItem {
     no_update_after: string | undefined;
     item_id: number;
+    studio_id: string;
```

Comment:
this JSONWarehouseItem object exactly represents the type in the smart contract.
We don't know the extent of meta data we're going to add to the item's metadata, so I'd suggest adding a `metadata: { [k: string]: string }` map to the item and storing the studio_id there.
Let's hold off on making these changes until we solidify the new smart contracts and let's discuss the best way of storing studio id on a token